tests/Integrations/Symfony/V4_2/CommonScenariosTest.php

```diff
@@ -54,13 +56,19 @@ public function provideSpecs()
                             'http.url' => 'http://localhost:9999/simple',
                             'http.status_code' => '200',
                             'integration.name' => 'symfony',
+                        ])
+                        ->withChildren([
+                            SpanAssertion::exists('symfony.kernel.handle')
+                                ->withChildren([
+                                    SpanAssertion::exists('symfony.kernel.request'),
+                                    SpanAssertion::exists('symfony.kernel.controller'),
+                                    SpanAssertion::exists('symfony.kernel.controller_arguments'),
+                                    SpanAssertion::exists('symfony.kernel.response'),
+                                    SpanAssertion::exists('symfony.kernel.finish_request'),
+                                ]),
                         ]),
-                    SpanAssertion::exists('symfony.kernel.handle'),
-                    SpanAssertion::exists('symfony.kernel.request'),
-                    SpanAssertion::exists('symfony.kernel.controller'),
-                    SpanAssertion::exists('symfony.kernel.controller_arguments'),
-                    SpanAssertion::exists('symfony.kernel.response'),
-                    SpanAssertion::exists('symfony.kernel.finish_request'),
+                    // This should be fixed as this should be a child
+                    // of the root span while here it is a lone trace
```

Comment:
Interesting. have we received reports for this?

Reply:
Nope, not so far.